configs/components/rubygem-puppet.rb

```diff
@@ -4,6 +4,8 @@
   pkg.version version
 
   case version
+  when '7.6.1'
```

Comment:
I think bolt is the only project that cares about this? We can probably just have a single 7 series version.